build error

## Error Info
```
cbits/random_initialized.c:9:9: error:
     error: macro names must be identifiers
        9 | #ifndef !DO_NOT_USE_GET_ENTROPY
          |         ^
  |
9 | #ifndef !DO_NOT_USE_GET_ENTROPY
  |         ^
cbits/random_initialized.c: In function ‘ensure_pool_initialized’:

cbits/random_initialized.c:51:9: error:
     error: macro names must be identifiers
       51 | #ifndef !DO_NOT_USE_GET_ENTROPY
          |         ^
   |
51 | #ifndef !DO_NOT_USE_GET_ENTROPY
   |         ^
`gcc' failed in phase `C Compiler'. (Exit code: 1)

```
## Env
- ghc: 8.10.7
- os: ubuntu 20.04
- gcc: 10.3.0

--- a/cbits/random_initialized.c
+++ b/cbits/random_initialized.c
@@ -6,7 +6,7 @@
 #include <unistd.h>
 
 #ifdef HAVE_GETENTROPY
-#ifndef !DO_NOT_USE_GET_ENTROPY
+#ifndef DO_NOT_USE_GET_ENTROPY
 static int ensure_pool_initialized_getentropy()
 {
     char tmp;
@@ -48,7 +48,7 @@ static int ensure_pool_initialized_poll()
 int ensure_pool_initialized()
 {
 #ifdef HAVE_GETENTROPY
-#ifndef !DO_NOT_USE_GET_ENTROPY
+#ifndef DO_NOT_USE_GET_ENTROPY
     if (ensure_pool_initialized_getentropy() == 0)
         return 0;
 #endif

